### PR TITLE
Release v2.18.1

### PR DESCRIPTION
**Pull request by zhiqiangjin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,72 @@
...
+  - New attributes ``egress_vpc_id`` and ``management_vpc_id`` in **aviatrix_firewall_instance**
+3. Implemented support for FireNet Keep Alive via Firewall LAN Interface:
+  - New attribute ``keep_alive_via_lan_interface_enabled`` in **aviatrix_firenet** resource and data source
+4. Implemented support for Gateway Certificate import:
+  - New resource **aviatrix_gateway_certificate_config**
+5. Implemented support for configuring AWS TGW CIDRs in **aviatrix_aws_tgw** using attribute ``cidrs``
+6. Implemented support for IKEv2 for route-based Site2Cloud connections in **aviatrix_site2cloud**
+7. Implemented support for ``metrics_only`` option in **aviatrix_datadog_agent**
+8. Implemented support for building OOB Transit/Spoke gateway and HA in different AZs/Subnets
+9. Implemented support for controller backend for AWSGOV, Azure, GCP and OCI providers
```
Comment (shuoliugit) [inline CHANGELOG.md]: typo - controller backup

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,72 @@
...
+10. Implemented support for attribute ``route_tables`` in **aviatrix_vpc** resource and data source
+11. Implemented support for Management Access form on-prem in **aviatrix_site2cloud**
```
Comment (shuoliugit) [inline CHANGELOG.md]: typo - from on-prem

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,72 @@
...
+  - ``zone``
+  - ``oob_management_subnet``
+  - ``oob_availability_zone``
+5. Updated following attributes to ForceNew in **aviatrix_aws_tgw**:
+  - ``tgw_name``
+  - ``aws_side_as_number``
+6. Updated following attributes to ForceNew in **aviatrix_aws_tgw_vpc_attachment**:
+  - ``tgw_name``
+  - ``vpc_id``
+7. Updated attribute ``local_as_number`` to ForceNew in **aviatrix_transit_gateway**:
```
Comment (shuoliugit) [inline CHANGELOG.md]: `local_as_number` is optional and computed, not ForceNew

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,72 @@
-## 2.18.0 (Unreleased)
+## 2.18.1 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.3.2355**
+- Supported Terraform version: **v0.12.x** and **v0.13.x**
+
+### Features:
+1. Implemented new resources to support TGW connect and connect peers:
+  - **resource_aviatrix_aws_tgw_connect**
+  - **resource_aviatrix_aws_tgw_connect_peer**
```
Comment (CyrusJavan) [inline CHANGELOG.md]: Don't need the resource_ prefix

**Review by mlin-aviatrix — APPROVED** [review]

**Review by shuoliugit — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,4 +1,72 @@
-## 2.18.0 (Unreleased)
+## 2.18.1 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.3.2355**
+- Supported Terraform version: **v0.12.x** and **v0.13.x**
+
+### Features:
+1. Implemented new resources to support TGW connect and connect peers:
```
Comment (inc1t3Ful) [inline CHANGELOG.md:7]: This is a feature, so should be TGW Connect and Connect peers. I can fix it alongside my doc PR

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,4 +1,72 @@
...
+4. Fixed an issue where upgrading controller causes CID to expire, which fails other functions in **aviatrix_controller_config**
+5. Fixed an issue where dot is not supported in ``spoke_vpc_id`` in **aviatrix_azure_spoke_native_peering**
+6. Fixed an issue where enabling encrypt volume with a customer managed keys fails in **aviatrix_gateway**, **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
+
+### Deprecations:
+1. Deprecated the in-line ``attached_vpc`` and ``attached_aviatrix_transit_gateway`` attributes in **aviatrix_aws_tgw**. Please use the standalone **aviatrix_aws_tgw_vpc_attachment** and **aviatrix_aws_tgw_transit_gateway_attachment** resources instead
+2. Deprecated the in-line ``transit_gw`` attribute in **aviatrix_spoke_gateway**. Please use the standalone **aviatrix_spoke_transit_attachment** resource instead
+3. Deprecated the in-line ``policy`` attribute in **aviatrix_firewall**. Please use the standalone **aviatrix_firewall_policy** resource instead
+4. Deprecated the in-line ``domain_names`` attribute in **aviatrix_fqdn**. Please use the standalone **aviatrix_fqdn_tag_rule** resource instead
+
```
Comment (inc1t3Ful) [inline CHANGELOG.md:67]: missing FireNet (`firewall_instance_association`). I will add into my docs PR

**Review by inc1t3Ful — APPROVED** [review]

**Pull request merged**